Sources/Core/DTOLoader.swift

```diff
@@ -0,0 +1,25 @@
+//  DTOLoader.swift
+
+import Foundation
+import Yams
+
+final class DTOLoader {
```

Comment:
or even `enum` as it's only namespace 
```suggestion
struct DTOLoader {
```